docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,278 @@ task.download_outputs()
+
+### Prerequisites
+
+Before running the simulation, ensure you have downloaded the input files. You
+can get them from the following
+[repository](https://develop.openfoam.com/committees/hpc/-/tree/develop/compressible/rhoPimpleFoam/LES/highLiftConfiguration).
+Once downloaded, move them into a directory named `highLiftConfiguration` inside
+your working folder.
+
+Here’s an example of how your directory structure should look after downloading
+the necessary files:
+
+```bash
+ls -lasgo highLiftConfiguration
+total 104
+ 0 drwxrwxr-x@  12     384 Jun 13 10:09 .
+ 0 drwx------@ 234    7488 Sep 19 15:15 ..
+ 0 drwxrwxr-x@  11     352 Jun 13 10:09 0.orig
+ 8 -rwxr-xr-x@   1     626 Jun 13 10:09 Allclean
+16 -rwxr-xr-x@   1    6998 Jun 13 10:09 Allrun
+ 8 -rw-rw-r--@   1     991 Jun 13 10:09 COPYING
+48 -rw-rw-r--@   1   21547 Jun 13 10:09 README.md
+ 0 -rw-rw-r--@   1       0 Jun 13 10:09 case.foam
+ 0 drwxrwxr-x@   5     160 Jun 13 10:09 constant
+ 0 drwxrwxr-x@  13     416 Jun 13 10:09 figures
+ 0 drwxrwxr-x@  27     864 Jun 13 10:09 system
+24 -rw-rw-r--@   1   11399 Jun 13 10:09 thumbnail.png
+```
+
+### Configuring and Running the Simulation
+
+We'll run the simulation on a virtual machine with 180 CPUs using the
+`c3d-standard-180` machine type. The simulation involves several preprocessing
+steps, including mesh generation (`blockMesh`, `snappyHexMesh`), and finally,
+the simulation run itself. 
+
+Below is the Python code to configure and run the OpenFOAM simulation:
+
+```python
+import inductiva
+import os
+
+machine_group = inductiva.resources.MachineGroup(machine_type="c3d-standard-180")
+machine_group.start()
+
+import inductiva
+
+# Set simulation input directory
+input_dir = "/path/to/highLiftConfiguration"
+
+# Read the simulation commands
+with open(os.path.join(input_dir,'input.txt'), 'r') as file:
+    commands = [line.strip() for line in file]
+
+# Initialize the Simulator
+openfoam = inductiva.simulators.OpenFOAM(distribution="esi")
+
+# Run simulation
+task = openfoam.run(
+    input_dir=input_dir,
+    commands=commands,
+    n_vcpus=128,
+    use_hwthread=True,
+    on=machine_group)
+
+# Wait for the task to finish and downloading the outputs
+task.wait()
+task.download_outputs()
+
+# Turn off the machine group
+machine_group.terminate()
+
+```
+
+The commands used in this simulation were taken from the `Allrun` script found
+in the downloaded files. Some adjustments were made because we execute all
+commands directly from the `highLiftConfiguration` directory, meaning we can't
+use `cd` to navigate into subdirectories and run commands from there.
+Additionally, every command must start with `runApplication` or `runParallel`,
+which is why even basic commands like `rm` and `mv` are preceded by `runApplication`.
+
+We have consolidated all the commands into a file named `input.txt`, with each
```

Comment:
I don't like having commands in a separate file. It just makes everything more verbose.
Apart from that, I think `input.txt` is not a good name for the commands file. Why not `commands.txt`?

Reply:
Changed it to `commands.txt`